doc/source/index.rst

```diff
@@ -1,116 +1,108 @@
-SciPy
-=====
+.. template taken from Pandas
 
-:Release: |release|
-:Date: |today|
+.. module:: scipy
 
-SciPy (pronounced "Sigh Pie") is open-source software for mathematics,
-science, and engineering.
+*******************
+SciPy documentation
+*******************
 
-.. toctree::
-   :maxdepth: 1
+**Date**: |today| **Version**: |version|
 
-   install_upgrade
-   api
-   release
+**Download documentation**: `PDF Version <https://docs.scipy.org/doc/scipy-1.6.1/scipy-ref-1.6.1.pdf>`__ | `Zipped HTML <https://docs.scipy.org/doc/scipy-1.6.1/scipy-html-1.6.1.zip>`__
 
-Tutorial
---------
+**Useful links**:
+`Binary Installers <https://pypi.org/project/scipy>`__ |
```

Comment:
This is not a very useful link, we should link to the most useful install doc page instead.